src/main/java/the_warlord/cards/warlord/parry_deck/DecisiveFactor.java

```diff
@@ -24,9 +26,9 @@ public DecisiveFactor() {
     @Override
     public void useParry() {
         if (this.upgraded) {
-            addToBot(new DrawCardAction(magicNumber));
+            addToBot(new ApotheosisAction());
```

Comment:
The upgraded version of this should certainly Purge, since it will do practically nothing when played more than once.